packages/terra-search-field/src/SearchField.jsx

```diff
@@ -231,11 +233,16 @@ class SearchField extends React.Component {
       ...customProps
     } = this.props;
 
-    const searchFieldClassNames = cx([
-      'search-field',
-      { block: isBlock },
+    const theme = this.context;
```

Comment:
Could we be using the themeContext hook?

Reply:
Synced with @pranav300.  didn't realize this wasn't a functional component when looking at the few lines that changed so we can't use the useContext hook here.